drift-transport-netty/src/main/java/io/airlift/drift/transport/netty/server/ThriftServerHandler.java

```diff
@@ -107,6 +107,10 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
         }
 
         log.error(cause);
+
+        if (ctx.channel().isActive()) {
```

Comment:
Do we need the `if` check?  Why not unconditionally close the context.

Reply:
I took it from the `io.netty.handler.ssl.SslHandler`. I'm not sure why it was added there. Maybe just to safe some cycles on Future creation.